package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "server-dev": "nodemon server/index.js",
-    "build-dev": "webpack --watch"
+    "build-dev": "webpack --watch",
```

Comment:
the "build-dev" script would usually have the option "-d source-map" so you can debug in the browser. 
the "build" script would omit this, sort of like you have here already.